### DISCUSSION TITLE
K1C cleaning nozzel before print qustion

### DISCUSSION THREAD

**Discussion by alkattaf** [Guilouz/Creality-Helper-Script-Wiki#452] [General]
![IMG_1504](https://github.com/Guilouz/Creality-K1-Series/assets/77775391/87df935e-8c7f-4afd-836b-457ee8a82e8a)

i have fixed same rubber of K1C plate to my k1 max plate but on k1 max the before print nozzle dosnt brush cuples time 

i have K1C and K1 max 
Both are rooted , i couldn't find any gcode regarding clear _nozzel_quick

The k1C brush the nozzel cuples time into rubber peace  , i want to make my k1 max to

**Comment by revunix** [comment]
Maybe this will help you: 
https://www.printables.com/model/803777-creality-k1c-silicone-nozzle-cleaner-mod-easy-bed
and also read this comment: 
https://www.printables.com/model/803777-creality-k1c-silicone-nozzle-cleaner-mod-easy-bed/comments/1653853

**Reply by TheWebMachine** [reply]
I was about to venture down the rabbit hole of 3rd party plates for my K1C...so glad to see this reply! Kudos! You probably saved me a few hours of hunting around. 🫡

(and 🤬 to Creality for still not releasing replacement PEI and textured plates for the K1C for purchase)

**Reply by PlerionPlorion** [reply]
Do you mean with the brush? If not, there are hundreds of replacement beds so idk what you mean, I have a creality textured PEI bed on my K1C right now.

**Reply by TheWebMachine** [reply]
Oh yeah, I already have 3rd party plates; love me some Fulament textured PEI. Sorry, I meant the rabbit hole of finding a solution for the brush situation. My fingers aren't nearly as fast as my squirrel brain. 🐿️

The fact that Creality doesn't already have replacement and textured plates _with brushes_ available to purchase is extremely annoying. These brushes aren't exactly made of adamantium and two teeth on my first plate's brush have already vanished. I figured I would eventually have to buy another Creality plate just to scavenge the brush. I'm not really wanting to mixing up a wee bit of silicone to make just a couple of brushes nor an easier-to-mix larger batch of silicone that ends up making a thousand brushes I'll never use. 🤣

That said, I have already printed the above bit and installed it. Using a fresh Fulament plate beautifully...until the current brush bites the dust, anyway. 

I am hoping a 3rd party starts selling replacement brushes solo in bulk. Though, I suspect Fulament will put out a plate with brush soon enough...they literally make plates for every size, type, and printer out there.

(Here's my squirrel brain kicking into overdrive: Maybe I **should** whip up a large batch of brushes and sell the surplus. Ugh...that sounds like work, though. 😎)

**Reply by PlerionPlorion** [reply]
How many prints have you done? I'm wondering if I should buy more brushes because I've only printed about 25 parts so far

**Reply by TheWebMachine** [reply]
Only a few dozen so far, but I do print with a lot of PETG. Do you have a source for just the brushes? I haven't found one.

**Reply by PlerionPlorion** [reply]
No but the printables page has a mold available as well, I'm hoping AliExpress will eventually obtain clones or officials since creality.

**Reply by TheWebMachine** [reply]
That was what I meant about making my own earlier...I may end up beating them to the punch by making a ton on my own and selling the excess.
